ingestion/src/metadata/data_quality/validations/table/pandas/tableCustomSQLQuery.py

```diff
@@ -28,4 +31,13 @@ class TableCustomSQLQueryValidator(
 
     def _run_results(self, sql_expression: str):
         """compute result of the test case"""
-        return self.runner[0].query(sql_expression)
+        import pandas as pd  # pylint: disable=import-outside-toplevel
+
+        try:
+            return pd.concat(runner.query(sql_expression) for runner in self.runner)
+        except MemoryError:
+            logger.error(
+                "Unable to compute due to memory constraints."
+                "We recommend using a smaller sample size or partitionning for the query."
+            )
+            return []
```

Comment:
You don't need to concatenante it here. https://github.com/open-metadata/OpenMetadata/pull/10603#discussion_r1157217665

Reply:
the query would return dataframe with or without rows here based on the query

Reply:
if the dataframes are empty, we concat, and the len of that dataframe being empty will result in test success

Reply:
Imagine a query that returns all the data. That will throw an error. We just need to sum the Len and not concatenate. Does it make sense?

Reply:
no, i tested this out, it will return dataframes with data, if the dataframe has data, we concat, and return the same, it will check the len of the rows.

but agree on the sum of length, that would reduce load on the memory, instead of concat, let me return the sum of len.